wger/weight/views.py

```diff
@@ -124,6 +130,74 @@ def export_csv(request):
     return response
 
 
+@login_required
+def import_weight_from_fitbit(request):
+    '''
+    Imports a user's data from fitbit
+    '''
+
+    FITBIT_CLIENT_ID = os.environ.get('FITBIT_CLIENT_ID')
+    FITBIT_CLIENT_SECRET = os.environ.get('FITBIT_CLIENT_SECRET')
+
+    server = FitbitOauth2Client(FITBIT_CLIENT_ID, FITBIT_CLIENT_SECRET)
+    authorization_url = server.authorize_token_url(
+        redirect_uri=("http://127.0.0.1:8000/en/weight/fitbit_sync/")
+    )[0]
+
+    if 'code' in request.GET:
+        code = request.GET.get('code')
+
+        headers = {
+            'Content-Type': 'application/x-www-form-urlencoded',
+            "Authorization": 'Basic ' + base64.b64encode(
+                FITBIT_CLIENT_ID.encode() + b':' + FITBIT_CLIENT_SECRET.encode()).decode()
+        }
+
+        data_params = {
+            'client_secret': FITBIT_CLIENT_SECRET,
+            'code': code,
+            'client_id': FITBIT_CLIENT_ID,
+            'grant_type': 'authorization_code',
+            'redirect_uri': 'http://127.0.0.1:8000/en/weight/fitbit_sync/'
+        }
+        response = requests.post(
+            server.request_token_url, data_params, headers=headers).json()
+
+        if "access_token" in response:
+            token = response['access_token']
+            headers['Authorization'] = 'Bearer ' + token
+            print('Fitbit data ', Fitbit.__dict__)
+
```

Comment:
Remove this print statement in the code.

Reply:
resolved, thanks